src/utils/breach-resolution.js

```diff
@@ -114,6 +114,7 @@ function appendBreachResolutionChecklist (userBreachData, options = {}) {
       const dataClasses = b.DataClasses
       const args = {
         companyName: b.Name,
+        breachedCompanyLink: `<a href="https://${b.Domain}" target="_blank">${b.Domain}</a>`,
```

Comment:
Issue, blocking: sometimes `.Domain` is an empty string. We'll probably want to either keep the previous string for those cases, or hide the entire suggestion in those cases.